include/net/mqtt.h

```diff
@@ -707,6 +707,18 @@ int mqtt_abort(struct mqtt_client *client);
  */
 int mqtt_live(struct mqtt_client *client);
 
+/**
+ * @brief Helper function to determine when next keep alive message should be
+ *        sent. Can be used for instance as a source for `poll` timeout.
+ *
+ * @param[in] client Client instance for which the procedure is requested.
+ *
+ * @return Time in milliseconds until next keep alive message is expected to
+ *         be sent. In case no keep alive is configured, function returns
```

Comment:
I think we could formulate the latter sentence slightly. I misunderstood it when reading it first time. Could we say here `Function will return UINT32_MAX if keepalive messages are not enabled.`

Reply:
Rephrased as suggested.